src/graphics/render_window.rs

```diff
@@ -389,6 +572,27 @@ impl RenderWindow {
     /// # Safety
     ///
     /// The cursor can not be destroyed while in use by the window.
+    ///
+    /// # Usage Example
+    ///
+    /// ```no_run
+    /// # use sfml::window::{Event, Style};
+    /// # use sfml::graphics::RenderWindow;
+    /// # // Create a new window with SFML window as name
+    /// # let mut window = RenderWindow::new((800, 600),
+    /// #                              "SFML window",
+    /// #                              Style::CLOSE,
+    /// #                              &Default::default());
+    /// # use sfml::window::{ Cursor, CursorType };
+    /// let cursor = Cursor::from_system(CursorType::Arrow);
+    /// if let Some(arrow_cursor) = &cursor {
+    ///     unsafe { window.set_mouse_cursor(arrow_cursor); }
+    /// }
+    /// // You need to ensure the SFML window closes before the cursor's end of life.
+    /// // Doing it the other way around will cause errouneous errors.
```

Comment:
Errouneous errors? That sounds a bit redundant. Would just "will cause errors" be enough?